tests/python_client/testcases/test_insert.py

```diff
@@ -2013,9 +2146,10 @@ def test_upsert_dataframe_using_none_data(self):
             "float_vector": vectors
         })
         collection_w.upsert(df)
-        exp = f"{ct.default_string_field_name} == ''"
```

Comment:
Need to add another case to keep this expr for query with None data which should return empty.